docs/generator_template.md

```diff
@@ -0,0 +1,575 @@
+
+      <Text indent={2} newLines={2}>
+        {`def __init__(self):
+            self.client = mqtt.Client()
+            self.client.connect(mqttBroker)`}
+      </Text>
+
+      <Text indent={2}>
+        <TopicFunction channels={asyncapi.channels().filterByReceive()} />
+      </Text>
+    </File>
+  )
+}
+
+```
+
+Run `npm test` on your terminal to ensure everything works as expected.
+
+In the next section, you'll add another channel to **asyncapi.yml** file called `temperature/dropped` and `temperature/risen` then run the template again to make sure it still works as expected.
+
+#### Update AsyncAPI document
+
+Update the AsyncAPI document to use two channels:
+
+```yml
+channels:
+  temperature/dropped:
+    description:  Notifies the user when the temperature drops past a certain point.
+    publish:
+      operationId: temperatureDrop
+      message:
+        description: Message that is being sent when the temperature drops past a certain point.
+        payload:
+          type: object
+          additionalProperties: false
+          properties:
+            temperatureId:
+              type: string
+
+  temperature/risen:
+    description: Notifies the user when the temperature rises past a certain point.
+    publish:
+      operationId: temperatureRise
+      message:
+        description: Message that is being sent when the temperature rises past a certain point.
+        payload:
+          type: object
+          additionalProperties: false
+          properties:
+            temperatureId:
+              type: string
+```
+
+And update your test script in test.py to test the two functions as below:
+
+```py
+    client.sendTemperatureDrop(randomId)
+    print("Temperature drop detected " + str(randomId) + " sent to temperature/dropped")
+    client.sendTemperatureRise(randomId)
+    print("Temperature rise detected " + str(randomId) + " sent to temperature/risen")
+```
+
+Run `npm test` to validate that everything works as expected. You should see logs similar to the snippet below in your terminal:
+
+```cmd
+Temperature drop detected 49040460 sent to temperature/dropped
+Temperature rise detected 49040460 sent to temperature/risen
+Temperature drop detected 66943992 sent to temperature/dropped
+Temperature rise detected 66943992 sent to temperature/risen
+```
+
+## Where to go from here?
+
+Great job completing this tutorial! You have learnt how to use an AsyncAPI file to create a Python MQTT template and used it with the Paho-MQTT library in Python to connect to an MQTT broker and publish messages.:]
+
+If you want to tinker with a completed template and see what it would look like in production, check out the [Paho-MQTT template](https://github.com/derberg/python-mqtt-client-template) developed by [Lukasz](https://github.com/derberg). You can also check out the accompanying [tutorial to create MQTT client code](https://derberg-github-io-git-articlecodegenpython-derberg.vercel.app/blog/asyncapi-codegen-python).
```

Comment:
```suggestion
If you want to tinker with a completed template and see what it would look like in production, check out the [Paho-MQTT template](https://github.com/derberg/python-mqtt-client-template/tree/v1.0.0). You can also check out the accompanying [article about creating MQTT client code](https://www.brainfart.dev/blog/asyncapi-codegen-python).
```